main_test.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2020 Google LLC
```

Comment:
We need to get https://github.com/GoogleCloudPlatform/traffic-director-grpc-bootstrap/pull/3 merged, before this.

Reply:
Ack.